Remotion/Web/Development.WebTesting.IntegrationTests/App.config

```diff
@@ -31,7 +31,7 @@
       cleanUpUnmatchedDownloadedFiles="false"
       requestErrorDetectionStrategy="Remotion.Web.Development.WebTesting.IntegrationTests.DiagnosticInformationCollectioningRequestErrorDetectionStrategy, Remotion.Web.Development.WebTesting.IntegrationTests">
     <hosting name="IisExpress" type="IisExpress" port="60401" />
-    <testSiteLayout rootPath=".\..\..\..\..\Development.WebTesting.TestSite">
+    <testSiteLayout rootPath=".\..\..\..\..\Development.WebTesting.TestSite.NetFramework">
```

Comment:
What is the root path? Shouldn't those be `.Shared`? Except for the root page/global.asax?

Reply:
That is the root path for the docker hosting. We did it this way for the object binding projects, but I don't know if it is possible to swap out the folders. Depends on how the build is started